ocaml/default.nix

```diff
@@ -405,6 +409,10 @@ with oself;
     };
   });
 
+  ff-pbt = osuper.ff-pbt.overrideAttrs (o: {
+    propagatedBuildInputs = o.propagatedBuildInputs ++ [ alcotest ];
```

Comment:
don't propagate a test dependency I guess? maybe add to buildInputs

Reply:
Yes :+1: